gtk/main.cc

```diff
@@ -127,12 +128,31 @@ int main(int argc, char** argv)
         config_dir = tr_getDefaultConfigDir(AppConfigDirName);
     }
 
+    auto lock = tr_get_lock_name(config_dir);
+    if (std::filesystem::exists(lock))
```

Comment:
We could write a PID to a lock file and check if a process with this PID exists also. What do you think?

Reply:
Until other issues are addressed, discussing file contents makes little sense. That is, unless you want to add more failure points because PIDs can be reused.